### PR TITLE
remove # prefixes when displaying shortcuts

### PR DESCRIPTION
**Pull request by gregquinn2001**
_Pull requests into Flux require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable (**N/A**) and :white_check_mark:._

Addresses JIRA 1373. no more hashtags when we render creator shortcuts. Still renders with same background color right now. Looking for thoughts on whether we think clinicians would want/expect the inserter shortcuts (record --> note) and creator shortcuts (record <-- note) to be differentiated visually.

Please post thoughts. I'm going to look at the concepts for styling our notes and see what they suggest.

## Submitter:

**Running the Application**

- [X] Manually tested in Chrome
- [ ] Manually tested in IE

**Documentation**

- [X] This pull request describes why these changes were made
- [X] Recognizes any potential shortcomings/bugs in the description 

**Code Quality**

- [x] 4-space indents - convert any tabs to spaces
- [x] Use public class field syntax for binding functions - ex. handleChange = () => { ... };
- [x] Code is commented

**Tests**

- [x] Existing tests passed

## Reviewer 1: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

## Reviewer 2: Name

- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by mgramigna** [comment]
[MG1] Backend tests are failing now. All seems to be a result of the same error: "Cannot read property 'signed' of null in FluxNotesEditor?

File: src/notes/FluxNotesEditor.jsx [hunk 1]
```diff
@@ -1494,7 +1494,7 @@ class FluxNotesEditor extends React.Component {
                     />
                     <div className='editor-content'>
                         <Slate.Editor
-                            className="editor-panel"
+                            className={this.props.selectedNote.signed ? "editor-panel" : "editor-panel in-progress-note"}
```
Comment (mgramigna) [inline src/notes/FluxNotesEditor.jsx]: MG1 stems from this line specifically

**Comment by Dtphelan1** [comment]
Boooooo -- ugh my changes must have broken it

**Comment by gregquinn2001** [comment]
I can fix it @Dtphelan1. You work on your task

**Comment by Dtphelan1** [comment]
.... Might have already fixed it. But I promise I'll work on my task :)

**Comment by gregquinn2001** [comment]
ok. I'll stand by then...

**Comment by Dtphelan1** [comment]
That said I'll leave [DP1] to you: 
FullApp test 6.1-6.12 fail -- they're looking for shortcuts using the old className ".structured-field" name, which needs to be updated to the new classes for both inserter and creator shortcuts.

**Comment by mgramigna** [comment]
Also worth mentioning, but doesn't warrant an MG. Now that we are using blue underlines in the editor and using the SASS variables defined based on the design guidelines, there is a slight mismatch in color with the underlines in the targeted data panel since those components' css files have not been converted to scss. When we are using SASS across the board, they will match up.

**Comment by gregquinn2001** [comment]
I'm working on DP1

**Comment by gregquinn2001** [comment]
DP1 resolved
